cassiopeia/preprocess/__init__.py

```diff
@@ -6,6 +6,8 @@
     align_sequences,
     call_alleles,
     collapse_umis,
+    error_correct_barcodes_to_whitelist,
```

Comment:
Are we worried that users will be confused by what `barcodes` means here, thinking it might mean `intBC` or `cellBC`?

Reply:
we can rename to `error_correct_cellbcs_to_whitelist`